Sources/Overload/OvDebug/include/OvDebug/Utils/ConsoleColor.h

```diff
@@ -87,4 +91,37 @@ namespace OvDebug
 		SetConsoleTextAttribute(hStdout, c.m_color);
 		return i;
 	}
+#else
+
+    inline std::ostream &blue(std::ostream &s) {
```

Comment:
We should stick with `{` on a new line instead of placing them at the end of a line for consistency purposes